lib/workers/repository/update/pr/changelog/source-github.ts

```diff
@@ -179,3 +181,27 @@ export async function getChangeLogJSON({
 
   return res;
 }
+
+function findTagOfRelease(
+  version: allVersioning.VersioningApi,
+  depName: string,
+  depNewVersion: string,
+  tags: string[]
+): string {
+  const regex = regEx(`(?:${depName}|release)[@-]`, undefined, false);
+  const excactReleaseRegex = regEx(`${depName}[@-_]v?${depNewVersion}`);
+  const exactTagsList = tags.filter((tag) => {
+    return excactReleaseRegex.test(tag);
+  });
+  let tagName: string = null;
```

Comment:
```suggestion
  let tagName: string | null | undefined = null;
```
ts null check

`.find` returns undefined